CO-2998

josueBulle opened the pull request:
I needed those PR
quentingigon/report_compassion_migration_v11
quentingigon/partner_communication_switzerland_migration_v11
quentingigon/payment_ogone_migration_v11
josueBulle/website_compassion

I also needed a fix for file upload:
https://github.com/CompassionCH/compassion-modules/pull/1288/

josueBulle committed:
website_event_compassion migration

josueBulle committed:
Various fixes

josueBulle committed:
CO-2972 sponsor should leave the treatment field empty

josueBulle committed:
CO-2973 New message when medical discharge is required

josueBulle committed:
CO-2974 add infos to registration creation message

josueBulle committed:
CO-2970 manually/automatically close events

josueBulle committed:
CO-2977 new button to see related invoices

josueBulle committed:
CO-2978 Auto change participant state to "attended"

josueBulle committed:
CO-2975 Generate invoices earlier

eicher31 reviewed (approved):
Just fix flake8 error.

josueBulle committed:
lints

josueBulle commented:
The `default_tickets` function was removed in odoo11. Now I think the default tickets are handled in the `event.type` model. But we use a wizard that (asks for the registration fees and) creates directly  the `event.event`. I did a quick fix but I'm not sure if I should go further and refactor 

https://github.com/odoo/odoo/commit/91617bfc78e3da94be2276fe1c82c4ce47eeb896

josueBulle committed:
_default_tickets was removed in odoo11

In Odoo, it is now the role of the event.type to handle the default tickets. But we have a wizard that create directly an event.event and edit the tickets
